tests/unit_test.py

```diff
@@ -66,6 +66,14 @@
     export_env,
 )
 
+from tests.unit_test_support import (
```

Comment:
I think you need to add an empty `__init__.py` file to `tests/` so that you can import it this way.